utils/api.ts

```diff
@@ -66,6 +66,16 @@ async function postRating(variables: EntityReview.AddVariables): Promise<boolean
   return !!res?.data
 }
 
+async function uploadImage(file: File): Promise<string> {
+  const formData = new FormData()
+  formData.append('file', file)
+  const res = await fetch(`${getImageBaseUrl()}/upload`, {
+    method: 'POST',
+    body: formData
+  }).catch((err) => { console.error(err); return null })
```

Comment:
formatting, please expand this to multiple lines if you start a new block. or alternatively don't return null as not returning anything will make it return undefined which is effectively the same here